pay-api/src/pay_api/models/invoice.py

```diff
@@ -46,6 +47,7 @@ class Invoice(db.Model, Audit, BaseModel):  # pylint: disable=too-many-instance-
 
     payment_line_items = relationship('PaymentLineItem')
     receipts = relationship('Receipt')
+    account = relationship(PaymentAccount)
```

Comment:
Any reason not to use a string identifier for PaymentAccount here instead?

Reply:
Sure, will do that

Reply:
Made those change in latest commit